### PR TITLE
Man fallback

### LINKED ISSUE #7: Windows support - help doesn't work
Running `elm-make --help` in Windows doesn't do anything, since it's shelling out to `man`, which is a *nix program.

### PR DESCRIPTION
**Pull request by tomekwi**
@bdukes, please look around to see if my changes make sense and if it fixes `--help` on Windows. If all is well, feel free to merge and close this PR 😃.

Fixes #7.

### CODE REVIEW

**Comment by tomekwi** [comment]
By the way, remember to run `bash -e scripts/manpages.sh` before testing `--help` – it’s normally done automatically via prepublish hook on my box before pushing a new version.

File: test.js [hunk 1]
```diff
@@ -41,6 +44,78 @@ test('Prints `--help`', (assert) => {
...
+    assert.is(command, 'man',
+      'tries to spawn `man`'
+    );
+
+    return { error: { code: 'ENOENT' } };
+  } };
+
+  const outputStream = qs((chunk) => {
+    assert.is(chunk, expectedHelpContent,
+      'prints elm-make output first'
```
Comment (bdukes) [inline test.js]: this message should reference the help output, right?

File: source/elm-live.js [hunk 2]
```diff
@@ -29,10 +30,22 @@ module.exports = (argv, options) => {
 
   // Display help
   if (args.help) {
-    spawnSync('man', [
+    const manProcess = spawnSync('man', [
       path.resolve(__dirname, '../manpages/elm-live.1'),
     ], { stdio: 'inherit' });
```
Comment (bdukes) [inline source/elm-live.js]: This causes the error to be output to the console, followed by the help text.  I think you'll need to set the [`stdio`](https://nodejs.org/api/child_process.html#child_process_options_stdio) option to ignore the error stream (and, I suppose, then write the error stream to `process.stderr` if the error isn't `'ENOENT'`).

**Comment by bdukes** [comment]
Running `bash -e scripts/manpages.sh` complains that `marked-man` isn't found.  I installed the npm package globally, and now it's back to complaining that `man` isn't found.  I added some dummy content to the fallback file for testing.

Also, `npm run manpages` fails with "'scripts' is not recognized as an internal or external command, operable program or batch file."  If I change the script to "bash -e scripts/manpages.sh" then it does call the script, with the same error as above.

I'm fine if this doesn't build on Windows, so long as it works when published.

**Comment by tomekwi** [comment]
Good point – `marked-man` is only available if you run the script via `npm run` – and `npm run` uses the default system shell so it can’t recognize the shebang. Right you are.

Can you please download the attached built file to `/manpages/elm-live.1.txt` and try running `elm-live --help` again?

[elm-live.1.txt](https://github.com/tomekwi/elm-live/files/195667/elm-live.1.txt)

File: test.js (hunk 1 above)
Comment (tomekwi) [inline test.js]: Yup, if I understand you well. Help output should be the same as the contents of `/manpages/elm-live.1.txt`.

File: source/elm-live.js (hunk 2 above)
Comment (tomekwi) [inline source/elm-live.js]: Yeah, good point. Sounds like more than a quick 5-minute-fix 😕. I’m afraid I won’t have time to fix it until Saturday. If you’re in a hurry, feel free to take over though 🙂.

**Comment by bdukes** [comment]
That txt file looks great!

**Comment by tomekwi** [comment]
Thanks 😊. That’s basically raw output from `man` piped into a file. Does `elm-live --help` work for you after you place the file where it belongs by hand?

**Comment by bdukes** [comment]
Yes, except for the error message about `man` at the top of it:

```
'man' is not recognized as an internal or external command,
operable program or batch file.
ELM-LIVE(1)                                                        ELM-LIVE(1)

NAME
       elm-live - A flexible dev server for Elm. Live reload included!

SYNOPSIS
       elm-live [...<options>] [--] ...<elm-make args>
       elm-live --help

DESCRIPTION
       First, we spawn elm-make with the elm-make args you’ve given.

       When  the  build is ready, we start a static HTTP server in the current
       directory. We inject a live reload snippet  into  every  HTML  file  we
       serve.  Every  time a static file has changed, we’ll reload your app in
       all browsers you’ve opened it with. (Mobile and IE included!)

       We also watch all *.elm files in the current directory and  its  subdi‐
       rectories.  Whenever  you  change,  add  or  remove  one of them, we’ll
       rebuild your program and reload the page.

OPTIONS
   --port=PORT
       Set the port to start the server at. If the port is  taken,  we’ll  use
       the  next  available  one. PORT should be a valid port number. Default:
       8000.

   --open
       We’ll open the app in your default browser as soon as the server is up.

   --no-recover
       When elm-make encounters a compile error, we keep elm-live running  and
       give  you  time  to  fix  your  code. Pass --no-recover if you want the
       server to exit immediately whenever it encounters a compile error.

   --help
       You’re looking at it.

EXAMPLES
       The simplest scenario:

         $ elm-live Main.elm --open

       Custom HTML file:

         $ echo \
           '<!doctype html>
           <link rel="stylesheet" href="/style.css" />
           <script src="/elm.js"></script>
           ' > index.html
         $ elm-live Main.elm --output=elm.js --open

SUPPORTED PLATFORMS
       We currently support all  Unix  systems.  We’re  also  doing  our  best
       https://github.com/tomekwi/elm-live/labels/Windows%20support  to  bring
       elm-live to Windows.

CREDITS
       Many thanks to Evan Czaplicki https://github.com/evancz, the creator of
       Elm,  for  git.io/elm-make https://git.io/elm-make – the most brilliant
       language compiler the world has ever seen! Without  elm-make,  elm-live
       would be a car without an engine.

       Many  thanks  to  Matt  DesLauriers https://github.com/mattdesl for the
       wonderful git.io/budo-server  https://git.io/budo-server.  That’s  what
       does the heavy lifting on the static server side.

LICENSE
       MIT ./License.md © Tomek Wiszniewski https://github.com/tomekwi

v2.0.6                            March 2016                       ELM-LIVE(1)

```

**Comment by tomekwi** [comment]
Alright, it seems to be ready now. Can you look through the changes and test it again please? If it works, feel free to merge and I’ll release ASAP.

**Comment by bdukes** [comment]
:+1: :shipit:

**Comment by tomekwi** [comment]
Great! I’ll :shipit: tomorrow or on Wendesday then.

**Pull request merged**